src/Components/test/E2ETest/Tests/PerformanceTest.cs

```diff
@@ -33,14 +33,12 @@ protected override void InitializeAsyncCore()
         public override Task InitializeAsync() => base.InitializeAsync(Guid.NewGuid().ToString());
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/32788")]
         public void HasTitle()
         {
             Assert.Equal("E2EPerformance", Browser.Title);
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/32788")]
```

Comment:
I'm least sure about the changes in this file. @wtgodbe were these quarantined simply due to problems w/ the test app combined w/ the older SDK❔

Mostly wondering: Should we leave these quarantined for 30 days after this gets in i.e. don't un-quarantine as part of this PR❔

Reply:
Thoughts on this❔ I feel like I'm violating @BrennanConroy's new process😉

Reply:
**Our** new process. There are always exceptions to the rule, this wasn't flaky before the SDK issues right?

Reply:
I'm struggling to remember.

Reply:
These should be unquarantined, the flakiness was due to the SDK